### DISCUSSION TITLE
connecting to a S7-200

### DISCUSSION THREAD

**Discussion by amarco** [gijzelaerr/python-snap7#382] [Q&A] [answered]
I am trying to connect to S7-200.
I  have the following snippet in python (using python-snap7 1.2 stable.

self.plc = snap7.client.Client() #create instance of client
self.plc.set_connection_params(ip, localtsap, remotetsap) #setup connection parameters
self.plc.connect() #try and connect

My problem is with the connect, it calls for using a ip address, slot and rack parameters which do not apply to the S7-200.

My question is how to connect to a device using the set_connection_params() method and not have the connect call require the restating of different connection parameters. I have not been able to find a connect option that would not take any parameters and use the info from the set_connection_params() call.

Marco

**Comment by swamper123** [comment, marked as answer]
Hi Marco,

[S7-200 is (normally) not in scope of snap7 since 2013. 
Anyway you can try connection to it as a LOGO device.](http://snap7.sourceforge.net/s7_200.html)

No guaranty that this will work, but otherwise you may have to look for another library.

**Comment by gijzelaerr** [comment]
Closing as stale. S7-200 is not supported by snap7. The suggestion to try connecting as a LOGO device was provided.
